README.md

```diff
@@ -1,6 +1,34 @@
 # notation-akv
 
-Azure Provider for Notation CLI
+Azure Provider for the Notary v2 [Notation CLI](https://github.com/notaryproject/notation)
+
+## Getting Started:
+The following summarizes the steps to configure the Azure Key Vault notation plugin, configure gatekeeper, sign and verify a container image to Azure Kubernetes Service
+
+```bash
+# Configure notation with the Azure Key Vault plugin
+notation plugin add azure-kv ~/.config/notation/plugins/azure-kv/notation-azure-kv
```

Comment:
Should we add steps on how to get the `notation-azure-kv` binary from the github release after the release is cut to this main readme?

Reply:
Yes, once the release is cut, I'll update.